test/jdk/java/awt/Focus/DeiconifiedFrameLoosesFocus/DeiconifiedFrameLoosesFocus.java

```diff
@@ -79,10 +79,12 @@ void test() {
         frame.setExtendedState(Frame.ICONIFIED);
 
         Util.waitForIdle(robot);
+        robot.delay(200);
```

Comment:
I remember some similar issues, so I 'grep" our tests. Looks like usually a bigger delay is used, like 500 or even 1 second.

Reply:
Normally we use delay of 1 sec after frame.setvisible and 500ms after events. 

Can you also please rectify the wildcard imports? Did you try to give CI job running this test for several iterations in all platforms, if yes, can you post it in JBS?